readthedocs/projects/tasks/builds.py

```diff
@@ -561,6 +561,9 @@ def run_setup(self):
             version=self.data.version,
             build=self.data.build,
             environment=self.get_vcs_env_vars(),
+            # Force the ``container_image`` to use one that has the latest
+            # ca-certificate package which is compatible with Lets Encrypt
+            container_image='readthedocs/build:ubuntu-20.04',
```

Comment:
you should be able to use the value from https://github.com/readthedocs/readthedocs.org/blob/274a377d5061df1cf8a457cec3f33d4f23b468f5/readthedocs/settings/base.py#L549-L553, instead of hardcoding it.